core-services/image-mirroring/performance-addon-operator/mapping_performance-addon-operator_quay

```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/ocp/performance-addon-operator:performance-addon-operator quay.io/performance-addon-operator-dev/performance-addon-operator:latest
+registry.svc.ci.openshift.org/ocp/performance-addon-operator:performance-addon-operator-registry-master quay.io/performance-addon-operator-dev/performance-addon-operator-registry:latest
```

Comment:
I just wonder if this should be called `-bundle-registry` instead of `-registry`. I don't really care myself, just checking about standards/conventions